core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java

```diff
@@ -120,14 +120,16 @@
    * locks and temporary files.
    *
    * @throws IOException if an I/O error occurs
+   * @throws InterruptedException if this thread is interrupted
    */
-  void sessionHeartbeat() throws IOException;
+  void sessionHeartbeat() throws IOException, InterruptedException;
```

Comment:
I don't see much value in exposing the `InterruptedException` if we just rethrow it afterwards.